engines/scumm/verbs.cpp

```diff
@@ -1046,12 +1043,26 @@ void ScummEngine_v7::drawVerb(int verb, int mode) {
 				}
 				--len;
 			}
+			if (_language == Common::HE_ISR) {
+				vs->curRect.right -= vs->curRect.left;
+				vs->curRect.left = _screenWidth - _charset->getStringWidth(0, tmpBuf);
+				vs->curRect.right += vs->curRect.left;
+			}
```

Comment:
Can you explain what you're doing here?

Reply:
Aligning the text to the right.
The first and third lines is replacing the old left position parameter with the new left position. (substract old, add new)

Reply:
Yes, but you're trying to right-align the text relative to the screen Is that intended? Does the original align the text to the left of the screen?

Reply:
Yes, this is the dialogue on SCUMM-7-8, which is aligned to the left.
altough I have checked this now again, the original version of COMI is indeed indented 5 pixels to the left (value of `vs->curRect.left`). (not 0 as I assumed)
ideally we would use this value to calculate
`drawLeft = _screenWidth - originalLeft - _widthOfString`
problem is the updated value is used for the clickable/hoverable mouse area, and it causes the new value to be received when drawing the next time, so using it make the text to jump from side to side.
the difference is (IMO) somewhat neglectable visually, though.
could be solved by making 2 `_verbs` arrays, but that is a change that may affect much more games.